### DISCUSSION TITLE
LDM Notes for May 15th, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8140] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-05-15.md

## Agenda

- `field` and `value` as contextual keywords
  - Usage in `nameof`
  - Should `value` be a keyword in a property or indexer get? Should `field` be a keyword in an indexer?
  - Should `field` and `value` be considered keywords in lambdas and local functions within property accessors?
  - Should `field` and `value` be keywords in property or accessor signatures? What about `nameof` in those spaces?
- Dictionary expressions

**Comment by huoyaoyuan** [comment]
The story may be different if special symbols are used for these contextual keywords, like `$value` `$field`.
We certainly don't go through this path, but it may be interesting to think about.

**Reply by 333fred** [reply]
We have no intention of doing that. We are explicitly moving forward with the breaking change version.
